internal/test/scorecard/tests.go

```diff
@@ -177,6 +178,10 @@ func CryostatRecordingTest(bundle *apimanifests.Bundle, namespace string, openSh
 	}
 	target, err := apiClient.Targets().Create(context.Background(), targetOptions)
 	if err != nil {
+		logErr := logWorkloadEvents(r, client, namespace, CryostatRecordingTestName)
+		if logErr != nil {
+			return fail(*r, fmt.Sprintf("failed to create a target: %s", logErr))
+		}
```

Comment:
Would it make sense to `defer` the `logWorkloadEvents`, but before deleting Cryostat CR?

Reply:
because currently, the error might appear as:

```
failed to create a target: <some-error-during-log>
```

so, its might not be clear which error causes the failed API request...